src/main/resources/db/migration/V4_CreateProject.sql

```diff
@@ -0,0 +1,13 @@
+CREATE TABLE project (
+    projectId INT NOT NULL AUTO_INCREMENT,
+    `name` VARCHAR(50) NOT NULL,
+    `value` DECIMAL(11, 2) NOT NULL,
+    techLeadId INT NOT NULL,
+    clientId INT NOT NULL,
+    salesEmployeeId INT NOT NULL,
+    startDate DateTime NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    finishDate DateTime,
```

Comment:
very pedantic but I think endDate works better here

Reply:
let leave at finishdate for now, matches prior schema